config/environments/development.rb

```diff
@@ -14,6 +14,11 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  config.active_record.verbose_query_logs = true
+
+  # Suppress logger output for asset requests.
+  config.assets.quiet = true
```

Comment:
and this?

Reply:
This looks like something that is set in most of all ours apps already (Rails default). If I remember right you just don't get asset requests in your Rails development server log which cuts out a lot of noise.